debbuild

```diff
@@ -1428,25 +1585,26 @@ sub binpackage {
 
     # Munge things so that Debian tools don't choke on errant blank lines
     $pkgdata{$pkg}{description} =~ s/\s+$//g;	# Trim trailing blanks
-    $pkgdata{$pkg}{description} =~ s/^ $/ ./mg;	# Replace lines consisting of " \n" with " .\n"
+    $pkgdata{$pkg}{description} =~ s/^\s+$/ ./mg;	# Replace lines consisting of " \n" with " .\n"
 
     # Give an estimate of the installation size
     my ($installedsize) =
       qx($specglobals{__du} -s --apparent-size $specglobals{buildroot}/$pkg) =~
         /(\d+)/;
 
+    my $maintainer = defined $pkgdata{main}{packager} ? $pkgdata{main}{packager} : expandmacros('%{_deb_maintainer}');
     my $control = "Package: $pkgdata{$pkg}{name}\n".
-	'Version: '.format_version($pkg)."\n".
-	( defined $pkgdata{$pkg}{group} ?
-	  "Section: $pkgdata{$pkg}{group}\n" : '' ).
-	"Priority: optional\n".
-	"Architecture: $pkgdata{$pkg}{arch}\n".
-	"Installed-Size: $installedsize\n".
-	( defined $pkgdata{main}{packager} ?
-	  "Maintainer: $pkgdata{main}{packager}\n" : '' ).
-	"Description: $pkgdata{$pkg}{summary}\n$pkgdata{$pkg}{description}\n".
-	( defined $pkgdata{main}{url} ?
-	  "Homepage: $pkgdata{main}{url}\n" : '' );
+                  'Version: '.format_version($pkg)."\n".
+                ( defined $pkgdata{$pkg}{group} ?
+                  "Section: $pkgdata{$pkg}{group}\n" : '' ).
+                  "Priority: optional\n".
+                  "Architecture: $pkgdata{$pkg}{arch}\n".
+                  "Installed-Size: $installedsize\n".
+                ( $maintainer ?
+                  "Maintainer: $maintainer\n" : '' ).
```

Comment:
Along with the comment around `%_deb_maintainer`, please revert this back to the way it was.